docker-local/README.md

```diff
@@ -94,10 +98,10 @@ docker-compose logs -f quoting-service
 docker-compose logs -f ml-api-adapter
 docker-compose logs -f central-settlement
 docker-compose logs -f account-lookup-service
-docker-compose logs -f dfspa-scheme-adapter dfspa-backend
-docker-compose logs -f dfspb-scheme-adapter dfspb-backend
+docker-compose logs -f dfspa-sdk-scheme-adapter dfspa-backend
+docker-compose logs -f dfspb-sdk-scheme-adapter dfspb-backend
 docker-compose logs -f transaction-requests-service
-docker-compose logs -f pisp-backend  pisp-scheme-adapter pisp-redis
+docker-compose logs -f pisp-backend  pisp-sdk-scheme-adapter pisp-redis
```

Comment:
maybe we should also add entries here to have pisp-thridparty-scheme-adapter-(inboud|outbound)

Reply:
done.